src/media/css/elements--header.styl

```diff
@@ -121,11 +122,13 @@ mkt-header-child {
     border: 0;
     border-radius: 5px;
     height: 100%;
-    min-width: $mobile-content-width + 10;
-    padding: 15px 0 15px 40px;
+    padding: 15px 40px;
     outline: none;
-    width: 50%;
+    width: 100%;
     z-index: 2;
+    +media--wide-tablet() {
```

Comment:
i might prefer the newline. maybe i don't like selectors sitting right next to rules without some separation. though it's almost a coinflip